rules/do-you-know-how-to-handle-diacritics/rule.md

```diff
@@ -0,0 +1,70 @@
+---
+type: rule
+archivedreason:
+title: Do you know how to handle diacritics?
+guid: 0d0c5084-5b42-45e0-afb5-c6f886d791ea
+uri: handling-diacritics
```

Comment:
Please update the folder path to match the uri `handling-diacritics`

Reply:
Done :)